File: MovieTix.Contracts/Models/VenueModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MovieTix.Contracts.Models
{
    public class VenueModel
    {
        public Guid VenueId { get; set; }
        public string EventName { get; set; }
        public string Address { get; set; }
        public int MaxRow { get; set; }
        public string MaxAisle { get; set; }
        public IEnumerable<TicketModel> Tickets { get; set; }
        //public List<List<TicketModel>> SeatingArrangement { get; set; }
    }
}

File: MovieTixApp/wwwroot/MovieWebApp/src/app/Venue/venue.model.ts
export class VenueModel {
  venueId: string;
  eventName: string;
  address: string;
  maxRow: number;
  maxAisle: string;
  tickets: TicketModel[];
}

export class TicketModel {
  reserved: boolean;
  purchased: boolean;
  selected: boolean;
  seat: SeatModel;
}

export class SeatModel {
  aisle: string;
  row: number;
}
